cage/cryptids/templates/cryptids/detail.html

```diff
@@ -3,13 +3,63 @@
 
 {% block title %}{{ animal.name }}{% endblock %}
 {% block CSS %}
-<!-- link for css -->
+<link rel="stylesheet" href="{% static 'cryptids/index.css' %}">
 {% endblock %}
 
 {% block content %}
+
+
+        <div class="tech-slideshow"> 
+        <div class="mover"> <img class="image2" src="./welshdragron.jpg">   </div>
```

Comment:
Here you could implement a for loop that automatically loops through the images of all animals in your database rather than hard coding them. I guess it depends what you want to do with this bit.